__tests__/app.test.js

```diff
@@ -129,6 +129,152 @@ describe("Using app.js to run the database of NC-games", () => {
         });
     });
   });
-  //Head describe DELETE later!
+  describe("6. PATCH/api/reviews/:reviews_id", () => {
+    test("200: response with a board game object with a updated vote", () => {
+      const review = { votes: 2 };
+      const review_id = 2;
+      return request(app)
+        .patch(`/api/reviews/${review_id}`)
+        .send(review)
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.review.votes).toBe(7);
+          expect(body.review).toHaveProperty("review_id", 2);
+          expect(body.review).toHaveProperty("title", "Jenga");
+          expect(body.review).toHaveProperty(
+            "review_body",
+            "Fiddly fun for all the family"
+          );
+          expect(body.review).toHaveProperty("designer", "Leslie Scott");
+          expect(body.review).toHaveProperty(
+            "review_img_url",
+            "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png"
+          );
+          expect(body.review).toHaveProperty("category", "dexterity");
+          expect(body.review).toHaveProperty("owner", "philippaclaire9");
+          expect(body.review).toHaveProperty(
+            "created_at",
+            "2021-01-18T10:01:41.251Z"
+          );
+        });
+    });
+    test("200: response with a board game object with a updated vote this time with 100 votes added", () => {
+      const review = { votes: 100 };
+      const review_id = 7;
+      return request(app)
+        .patch(`/api/reviews/${review_id}`)
+        .send(review)
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.review.votes).toBe(109);
+          expect(body.review).toHaveProperty("review_id", 7);
+          expect(body.review).toHaveProperty(
+            "title",
+            "Mollit elit qui incididunt veniam occaecat cupidatat"
+          );
+          expect(body.review).toHaveProperty("category", "social deduction");
+          expect(body.review).toHaveProperty("designer", "Avery Wunzboogerz");
+          expect(body.review).toHaveProperty("owner", "mallionaire");
+        });
+    });
```

Comment:
This is the same test as the one above, we can remove one of these

Reply:
Okay, I did the second test to be above the simpler test, however if you believe they are the same then all is good.